### DISCUSSION TITLE
Compiling under Windows - missing compile.bat

### DISCUSSION THREAD

**Discussion by electrickite** [pawelsalawa/sqlitestudio#4159] [General]
The instructions for compiling under Windows mention running the `compile.bat` script in the `sqlitestudio\scripts\win32` directory. This script seems to be missing from both the master and 3.3 branches. There is a `compile.tcl` script, but no instructions about how to run this script under Windows. Any advice?

**Comment by pawelsalawa** [comment]
You can download this Tcl (single file) interpreter: https://tclkits.rkeene.org/fossil/raw/tclkitsh-8.6.3-win32-ix86.exe?name=037adc95892780f49f6a1fccb04d33bbe9e2ae84
And run it like (assuming you're in the folder where compile.tcl file is):
```
C:\path\to\tclkitsh-8.6.3-win32-ix86.exe compile.tcl
```
